soaks/tests/splunk_hec_indexer_ack_blackhole/terraform/splunk_hec_gen.toml

```diff
@@ -0,0 +1,14 @@
+worker_threads = 2
```

Comment:
Is this one used anywhere? I don't see any references to it.

Reply:
Ah no, this is a legacy file I had from prior testing. In fact, lading no longer supports `toml` files. Thanks for catching, will remove